src/App.css

```diff
@@ -1,8 +1,231 @@
-.container {
-  border: 1px solid black;
-  padding: 1rem;
+:root {
+  --azure: #0290ff;
+  --white: #fff;
+  --black: #121212;
 }
 
-li {
+* {
+  box-sizing: border-box;
+  margin: 0;
+  border: none;
+  padding: 0;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+}
+
+.navContainer {
+  width: 100%;
+  height: 5.938rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: var(--white);
+  display: flex;
+  align-items: center;
+  max-width: 100vw;
+}
+
+.navContainer h2 {
+  margin: 0 2.938rem 0.125rem 0;
+  font-size: 1.875rem;
+  color: var(--azure);
+}
+
+.linksUl {
+  display: flex;
+  align-items: center;
+  gap: 2.563rem;
   list-style: none;
 }
+
+.linksUl a {
+  color: var(--black);
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+}
```

Comment:
- [ ] The avatar icon is still not aligned properly, I'd suggest using `justify-between` to push the two-child div into the proper place.